pkg/controller/consoleservice/consoleservice_controller.go

```diff
@@ -571,6 +571,12 @@ func applyDeployment(consoleservice *v1beta1.ConsoleService, deployment *appsv1.
 
 		install.ApplyEnvSimple(container, "OPENSHIFT_AVAILABLE", strconv.FormatBool(util.IsOpenshift()))
 
+		if consoleservice.ObjectMeta.GetAnnotations() != nil {
+			install.ApplyEnv(container, "ITEM_REFRESH_RATE", func(envvar *corev1.EnvVar) {
+				envvar.Value = consoleservice.ObjectMeta.GetAnnotations()["refreshRate"]
```

Comment:
EnMasse internal annotations should be prefixed with `enmasse.io/` to keep them clear of the user's domain.   I infer from AnnotationKeys.java that our preferred convention is hyphen separated rather than camelCase, so I would go `enmasse.io/console-refresh-rate`.